libkbfs/md_journal_test.go

```diff
@@ -162,6 +170,36 @@ func checkIBRMDRange(t *testing.T, uid keybase1.UID,
 	}
 }
 
+func BenchmarkMDJournalBasicMDv2(b *testing.B) {
+	_, _, id, signer, ekg, bsplit, tempdir, j :=
+		setupMDJournalTest(b)
+	defer teardownMDJournalTest(b, tempdir)
+
+	mdCount := 10
+	revision := MetadataRevision(10)
+	prevRoot := fakeMdID(1)
+	for i := 0; i < b.N; i++ {
```

Comment:
It probably makes sense to do:

```go
b.ResetTimer()
defer b.StopTimer()
```

right before this loop, so we don't include test setup in the benchmark timings.